### DISCUSSION TITLE
Best pizzeria ever

### DISCUSSION THREAD

**Discussion by Muis903** [Muis903/pizzeria#7] [Ideas]
All ideas are welcome!

**Comment by Muis903** [comment]
What is the best following of file development?
1. Order, Checkout, Paying, Kitchen, data,  main
2. Checkout, Kitchen, Order, Paying, data, main
3. Doesn't matter...
4. Your own opinion -

**Reply by sevamove** [reply]
4. It does not really matter at the present state of development. Firstly, I prefer to define what type of libraries we are gonna to implement in this project and what is the idea. The consumer interacts first with the cashier, then the cashier uses the software where he puts in the client's `orders`, after that calculates the total `price` and process the `payment`. ﻿Subsequently, the `order` goes through the `kitchen` and comes back to the `checkout` to give it away to the client.
